libraries/Microsoft.Bot.Builder.Azure/MsiServiceCrendtial.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.Services.AppAuthentication;
+using Microsoft.Bot.Connector.Authentication;
+using Microsoft.Rest;
+
+namespace Microsoft.Bot.Builder.Azure
+{
+    public class MsiServiceCrendtials : ServiceClientCredentials
```

Comment:
>MsiServiceCrendtials [](start = 17, length = 20)

spelling of credentials

Reply:
comment describing class

---
In reply to: [302186841](https://github.com/microsoft/botbuilder-dotnet/pull/2206#discussion_r302186841) [](ancestors = 302186841)